torch/nn/quantized/modules/conv.py

```diff
@@ -200,7 +202,7 @@ def from_float(cls, mod):
         wt_scale, wt_zp = weight_observer.calculate_qparams()
         bias_scale = (wt_scale * act_scale).float()
         qweight = torch.quantize_linear(
-            mod.weight.float().permute([0, 2, 3, 1]).contiguous(),
+            mod.weight.float().contiguous(),
```

Comment:
I think we can remove contiguous as well

Reply:
Let's do this later. I want to get this landed because it's blocking a customer team. The extra contiguous() should be a no-op anyway